src/test-utils/render.js

```diff
@@ -25,14 +31,12 @@ export function TestWrapper({
 }
 
 export function render(ui, options = {}) {
-    const queryClient = new QueryClient()
-
     return renderOrig(ui, {
         ...options,
         wrapper: ({ children }) => (
-            <TestWrapper wrapper={options.wrapper} queryClient={queryClient}>
-                {children}
-            </TestWrapper>
+            <DataProvider baseUrl="http://dhis2-tests.org" apiVersion="39">
```

Comment:
I've had to make a couple of small changes for the test-utils. The query client needed to be inside a Data Provider in order to work, so I added it here .. passing it as `wrapper` property didn't work because in `TestWrapper`, we're rendering the `content` then injecting it to the wrapper so the content doesn't have access to the provider. I could change that logic instead but this looked more straightforward; leaving DataProvider for tests that need it and it won't affect those that don't.

Reply:
I'm wondering if it'd make more sense to use the `CustomDataProvider` here. But that's a discussion we'll have to defer until we talked about how to proceed with msw

Reply:
ok, I can use CustomDataProvider then and will keep the msw tests separately for a discussion. Switching the tests to CustomDataProvider would help me as well to see what works and doesn't in practice compared to msw